### DISCUSSION TITLE
su-dung-python-de-mo-phong-bai-toan-xac-suat-monty-hall/

### DISCUSSION THREAD

**Discussion by giscus** [USTH-Coders-Club/USTH-Coders-Club.github.io#31] [Announcements]
# su-dung-python-de-mo-phong-bai-toan-xac-suat-monty-hall/

Bài toán xác suất kinh điển Monty Hall một thời đã thách thức hàng ngàn bộ não trên thế giới, từ học sinh cấp 3 cho đến những giáo sư và các chuyên gia phân tích. Tính đến ngày nay, đã có rất nhiều lời giải thích và đáp án cho câu đố này dưới góc độ toán học xuất hiện trên Internet, chủ yếu sử dụng kiến thức về xác suất thống kê. Hôm nay, UCC sẽ giới thiệu cho các bạn một hướng tiếp cận khác, đó là sử dụng ngôn ngữ lập trình Python để mô phỏng lại tình huống bài toán, từ đó đưa ra lời giải thuyết phục nhất từ số liệu đã thu được.

https://usth-coders-club.github.io/su-dung-python-de-mo-phong-bai-toan-xac-suat-monty-hall/

**Comment by Ph1eu** [comment]
bài này ok
